src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -266,5 +294,11 @@ public bool OnDeepLink(string deepLink)
             handler(deepLink);
             return true;
         }
+
+        public void OnInboxUpdated()
+        {
+            var handler = onInboxUpdatedReceived;
+            handler();
```

Comment:
null check?

Reply:
The idiomatic way of invoking the handler while checking for null is this:

```
onMessageCenterUpdated?.Invoke()
```

If the hander is typed as EventHandler, then you invoke it like this:

```
onMessageCenterUpdated?.Invoke(this, EventArgs.Empty);
```

Reply:
A null check isn’t really needed, as the only way this call happens is when there is an event handler. However, it certainly doesn’t hurt to do the null check.

Reply:
It's good practice to check for null when calling event handlers.